recipes/qt/5.x.x/conanfile.py

```diff
@@ -209,6 +209,8 @@ def configure(self):
         if tools.os_info.is_linux:
```

Comment:
This should use `self.settings.os`.
```suggestion
        if self.settings.os in ("FreeBSD", "Linux"):
```